ui-test/allTestsSuite.ts

```diff
@@ -0,0 +1,87 @@
+import { VSBrowser, WebDriver, SideBarView, ActivityBar, EditorView, By, WebView, TextEditor, Workbench, NotificationType } from "vscode-extension-tester";
+import { expect } from 'chai';
+import { addFolderToWorkspace, removeFolderFromWorkspace } from "./common/commonUtils";
+let path = require('path');
+import { pypiWithVulnsUITest } from './manifestFileTests/pypiWithVulns';
+import { pypiWithoutVulnsUITest } from './manifestFileTests/pypiWithoutVulns';
+import { npmWithVulnsUITest } from './manifestFileTests/npmWithVulns';
+import { npmWithoutVulnsUITest } from "./manifestFileTests/npmWithoutVulns";
+var fs = require('fs');
+let os = require('os');
+import debug from 'debug'
+const log = debug('server');
+import { delay } from "./common/helperUtils";
+import { goWithoutVulnsUITest } from "./manifestFileTests/goWithoutVulns";
+import { mavenWithoutVulnsUITest } from "./manifestFileTests/mavenWithoutVulns";
+import { goWithVulnsUITest } from "./manifestFileTests/goWithVulns";
+import { mavenWithVulnsUITest } from "./manifestFileTests/mavenWithVulns";
+import { checkPIEBTN, checkStatusBarBTN } from "./common/negativeTests";
+var assert = require('assert');
+
+describe('Fabric8-Analytics-VSCode-Extension UI tests', function () {
+    let driver: WebDriver;
+    let homedir: string;
+    let dir = path.resolve("./manifests");
+    before(async function () {
+        driver = VSBrowser.instance.driver;
+        homedir = dir
+        const control = await new ActivityBar().getViewControl('Explorer');
+        const explorerView = await control.openView();
+        addFolderToWorkspace(homedir);
+    });
+
+    describe('checking workspace section', function () {
+        it('workspace sections should not be null for manifests', async function () {
+            this.timeout(5000);
+            const sections = new SideBarView().getContent().getSections();
+            expect(sections).not.to.be.null;
+        })
+    });
+
+    describe('negative test 1 : statusbar btn absent before opening manifest file', checkStatusBarBTN)
+
+    describe('pypi tests with vulns', pypiWithVulnsUITest)
+    describe('npm tests with vulns', npmWithVulnsUITest)
+    describe('go tests with vulns', goWithVulnsUITest)
+    describe('maven tests with vulns', mavenWithVulnsUITest)
+
+    describe('change workspace', function () {
+        it('remove manifests folder from workspace and add manifest1 folder', async function () {
```

Comment:
Dont add test implementations in this file only add definations

Reply:
@sawood14012 I have done this way, PTAL here https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/527/commits/2cafd70f69ef49fd87787c3478aaa07e68c9773a#:~:text=describe('test%20with%20vulns'%2C%20testwithvulns)%3B 
Is it good or should I just make function call without using describes (as it won't be a good idea to add functions at same level as describes)?

Reply:
why do you think its not a good idea ?

Reply:
let me do that way

Reply:
as I am using multilevel describes so I think calling a function directly without describe won't follow good mochajs code writing approach

Reply:
i think you didnt understood me i said is to use describe but the describe block which has a function put it in a different file and import 
`describe('change workspace, ChangeWorkspace);`

Reply:
yes, I was thinking about line no 46 (which refers to test with vulns). but now get it